### ISSUE TITLE
不能识别英文换行符

### ISSUE THREAD

**Issue by pikakolendo02** [pdfcopy/pdfcopy.github.io#2]
比如有一段英文：
(i) All representative
compound-points of the test set in the **multidimen-
sional** descriptor space must be close to those of the
training set. 
转换过来后变成：
(i) All representative compound-points of the test set in the **multidimen- sional** descriptor space must be close to those of the training set.
